src/HealthGPS/analysis_module.cpp

```diff
@@ -304,6 +305,47 @@ DALYsIndicator AnalysisModule::calculate_dalys(Population &population, unsigned
                           .disability_adjusted_life_years = yll + yld};
 }
 
+void AnalysisModule::calculate_population_statistics(RuntimeContext &context) const {
+    auto num_factors_to_calculate =
+        context.mapping().entries().size() - factors_to_calculate_.size();
+
+    for (const auto &entity : context.population()) {
```

Comment:
Maybe call this variable `person` rather than `entity`? It's not your fault -- it looks like there are loads of `entity`s in this file (and throughout the code base) -- but it did take me a while to twig that it referred to a person